packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -704,13 +704,32 @@ export const sampleFactory = (model: ModelData): string[] => [
 	`python -m sample_factory.huggingface.load_from_hub -r ${model.id} -d ./train_dir`,
 ];
 
+function get_widget_examples_from_st_model(model: ModelData): string[] | undefined {
+	const widgetExample = model.widgetData?.[0] as WidgetExampleSentenceSimilarityInput | undefined;
+	if (widgetExample) {
+		return [widgetExample.source_sentence, ...widgetExample.sentences];
+	}
+}
+
 export const sentenceTransformers = (model: ModelData): string[] => {
 	const remote_code_snippet = model.tags.includes(TAG_CUSTOM_CODE) ? ", trust_remote_code=True" : "";
+	const exampleSentences = get_widget_examples_from_st_model(model) ?? [
+		"The weather is lovely today.",
+		"It's so sunny outside!",
+		"He drove to the stadium.",
+	];
 
 	return [
 		`from sentence_transformers import SentenceTransformer
 
-model = SentenceTransformer("${model.id}"${remote_code_snippet})`,
+model = SentenceTransformer("${model.id}"${remote_code_snippet})
+
+texts = ${JSON.stringify(exampleSentences, null, 4)}
+embeddings = model.encode(texts)
```

Comment:
```suggestion
sentences = ${JSON.stringify(exampleSentences, null, 4)}
embeddings = model.encode(sentences)
```

in the [docs here](https://sbert.net/docs/quickstart.html), var name `sentences` is used

Reply:
Indeed. The reason that I chose `text` originally is because technically, you can also feed texts that aren't sentences into a model, like paragraphs. But I'll use `sentences` to align with the docs.